cfme/utils/apidoc.py

```diff
@@ -39,11 +40,13 @@ def purge_module_apidoc(sphinx, exception):
         return
 
     try:
-        sphinx.info(bold('cleaning autogenerated docs... '), nonl=True)
+        # sphinx.info(bold('cleaning autogenerated docs... '), nonl=True)
```

Comment:
should these be removed?

Reply:
Yeah, I think they should just be removed, with the change in Sphinx version, they result in an `AttributeError`. But I'm not 100% sure if I should change them to print statements or just remove them altogether

Reply:
I changed them to logger calls, as that is what `Sphinx` 2.1.2 uses